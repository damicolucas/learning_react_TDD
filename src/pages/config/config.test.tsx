import React from "react";
import { RecoilRoot } from "recoil";
import { render } from "@testing-library/react";
import Config from ".";


const mockNavigator = jest.fn()

jest.mock('react-router-dom', () => {
    return {
        useNavigate: () => mockNavigator
    }
})

describe("the config page", () => {
  test("must be correct compiled", () => {
    const { container } = render(
      <RecoilRoot>
        <Config />
      </RecoilRoot>
    );
    expect(container).toMatchSnapshot();

  });
});
