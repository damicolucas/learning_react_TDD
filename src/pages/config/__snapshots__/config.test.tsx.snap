// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the config page must be correct compiled 1`] = `
<div>
  <div
    class="sc-bczRLJ fSSXVE"
  >
    <section>
      <h2>
        Vamos come√ßar!
      </h2>
      <form>
        <div
          class="sc-dkzDqf jzFXIQ"
        >
          <input
            placeholder="Enter players names"
            type="text"
            value=""
          />
          <button
            disabled=""
          >
            Submit
          </button>
        </div>
        
      </form>
      <ul />
      <footer
        class="sc-gsnTZi qDuGV"
      >
        <button
          disabled=""
        >
          Iniciar Brincadeira
        </button>
        <img
          alt="shopping bag"
          src="/img/sacolas.png"
        />
      </footer>
    </section>
  </div>
</div>
`;
